src/pages/PassportRedemption/GiveawayPopover.tsx

```diff
@@ -1,121 +1,131 @@
-import React , { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
-import GreenCheck from './GreenCheck'
-import InstagramEnabled from './InstagramEnabled'
+import GreenCheck from './GreenCheck';
+import InstagramEnabled from './InstagramEnabled';
 import InstagramDisabled from './InstagramDisabled';
 
-import { getAllParticipatingSellers, getGiveawayTicketsForContact } from '../../utilities/api/interactionManager'
-
+import {
+  getAllParticipatingSellers,
+  getGiveawayTicketsForContact,
+} from '../../utilities/api/interactionManager';
 
 interface Props {
-  showInstagram : boolean
-  contactId: string
-}
-
-interface TextProps {
-  icon ?: boolean
+  showInstagram: boolean;
+  contactId: string;
 }
 
-interface IgButtonProps {
-  disabled ?: boolean
-}
 interface GiveawayData {
-  weekly_giveaway_entries: String
-  number_of_visits_left : undefined | number
+  weekly_giveaway_entries: String;
+  number_of_visits_left: undefined | number;
 }
 
 const GiveawayPopover = (props: Props) => {
-  const [showInfo, setShowInfo] = useState(false);
+  const [showInfo, setShowInfo] = useState(true);
   const [giveawayData, setData] = useState<GiveawayData>({
     weekly_giveaway_entries: '',
-    number_of_visits_left: undefined
-  })
+    number_of_visits_left: undefined,
+  });
+
+  async function setGiveawayCount(contactId: string) {
+    const { data: allSellers } = await getAllParticipatingSellers();
+    const {
+      data: { weekly_giveaway_entries, unique_seller_tickets },
+    } = await getGiveawayTicketsForContact(contactId);
+    setData({
+      weekly_giveaway_entries,
+      number_of_visits_left: allSellers.length - unique_seller_tickets,
+    });
+  }
```

Comment:
Did we end the component early?

Reply:
Yeah @justintmckibben told me I could merge it in to get it out, and then submit another PR with refactors